multidoc/make.jl

```diff
@@ -56,8 +80,9 @@ MultiDocumenter.make(
 
 # Deploy to Github with running as a Github action
 if haskey(ENV, "GITHUB_ACTIONS")
-    isdir("multidoc-preview") || mkdir("multidoc-preview")
-    multidocroot = normpath(joinpath(@__DIR__, "../multidoc-preview/"))
+    output_dir = "library"
+    isdir(output_dir) || mkdir(output_dir)
+    multidocroot = normpath(joinpath(@__DIR__, "../", output_dir))
```

Comment:
Should be sufficient:
```suggestion
    multidocroot = normpath(joinpath(@__DIR__, "..", output_dir))
```

Reply:
I fixed these issue in [96abc8c](https://github.com/TuringLang/turing.ml/commit/96abc8c744782c6c566080128370b0f7a6c425e3)